docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,188 @@
+
+Both new components will implement the already existing pattern that components implement (e.g. OpenTelemetry Operator, Prometheus Operator).
+That being an implementation of the `Deployer` interface as well as an additional step the the `Seed` reconciliation flow.
+
+#### Access to VictoriaLogs in the `Garden` cluster
+
+Currently, Vali access is behind a service in the `garden` namespace.
+This pattern will remain and log shippers will only be expected to use the `OTLP` protocol, instead of the `Loki` protocol, via HTTP when inserting logs to VictoriaLogs.
+
+### 2. Deployment of VictoriaLogs on Seed Clusters
+
+Analogously to the deployment in the `Garden` cluster, this includes 2 steps:
+- Deployment of a new [VictoriaOperator](https://github.com/VictoriaMetrics/operator) component during the `Seed` reconciliation flow in the `Garden` namespace.
+- Deployment of a new `VictoriaLogs` k8s deployment during the Seed reconciliation flow but *after* the `VictoriaOperator` deployment has finished.
+This component will include a `VLSingle` CustomResource that the operator will reconcile in the `garden` namespace.
+
+#### Access to VictoriaLogs in the `Seed` cluster
+
+Since the setup is the same as in the `Garden` cluster, see details in 'Access to VictoriaLogs in the `Garden` cluster'.
+
+### 3. Deploy VictoriaLogs in Shoot control plane
+
+During the `Shoot` reconciliation flow, the new `VictoriaLogs` component will be deployed to the control plane namespace of `Shoot` clusters.
+The operator will manage the resulting `VLSingle` CustomResource and create a `VictoriaLogs` k8s deployment.
+
+Example `VLSingle` CR:
+```
+apiVersion: operator.victoriametrics.com/v1
+kind: VLSingle
+metadata:
+  name: example
+  namespace: shoot--local--local
+spec:
+  storage:
+    resources:
+      requests:
+        storage: 30Gi
+  resources:
+    requests:
+      memory: 100Mi
+      cpu: 10m
+    limits:
+      memory: 300Mi
+      cpu: 50m
+```
+
+#### Access to VictoriaLogs in the `Shoot` control plane
+
+Access to Vali is exposed via an ingress in the shoot control-plane.
+This will continue with VictoriaLogs as well.
+Log shippers will need to be configured via OTLP.
+
+### 4. Migration from Vali to VictoriaLogs
+
+#### Data Migration
+During all the replacements of `Vali` we want to ensure that existing clusters do not get disrupted.
+For this reason, 3 strategies for migration have been considered:
+- "One-shot" migration.
+Reformat all the Vali data into the VictoriaLogs format (or OTLP).
+Could be done via the OpenTelemetry Collector since there is no tool that exists to do this OOTB.
+    Pros:
+    - Migration step is quick.
+    - No friction during the migration period since there is only one source of logs after the migration.
+    Cons:
+    - Development of the necessary tooling to do the migration.
+    - Risk of data loss if something goes wrong during the migration.
```

Comment:
The rendered output of the pros/cons does not look good, e.g. cons is on the same line as the last pro bullet.
Please check if adding line breaks helps. (This also applies to the following sections.)